Dockerfile

```diff
@@ -35,9 +38,6 @@ FROM base
 
 COPY --from=builder /go/bin/starport /usr/bin
```

Comment:
What about we move them above this line (because this is the actual image will be used by the end users, and we might migrate the base image to alphine)?

Reply:
@ilgooz I'll try it tonight

Reply:
@ilgooz sorry, there's a lot going on in the attached screenshot, but it seems that the permissions issue persists if I put the `USER` line there

![image](https://user-images.githubusercontent.com/3329959/122027637-48ccbe00-cd99-11eb-90d2-2a06b4c09d4b.png)

Reply:
Oh okay, can we also try setting a chmod with "RUN chmod 744 (or other)" after the "USER" param to see if it'll fix it? Otherwise, we can keep things as before.

Reply:
Thanks for checking it out!